typings/tests/main-test.tsx

```diff
@@ -60,7 +60,12 @@ const HookTest = () => {
     referenceElement,
     popperElement,
     {
-      modifiers: [{ name: 'arrow', options: { element: arrowElement } }],
+      modifiers: [
+        {
+          name: 'arrow',
+          options: { element: arrowElement as HTMLElement | undefined },
```

Comment:
I just released `@popperjs/core@2.3.3` to allow the arrow element to be `null`, you should be able to get rid of this `as` thingy